docs/.vuepress/config.js

```diff
@@ -306,9 +306,9 @@ module.exports = {
       }
     },
     nav: [
-      { text: 'API', link: 'https://paka.dev/npm/xstate/' },
+      { text: 'API', link: 'https://doc.deno.land/https://esm.sh/v131/xstate' },
```

Comment:
The paka thing feels lighter to me and less daunting. It also seems to favor JSDoc a little bit more (often you can find the JSDoc comments earlier on the displayed page) and displays some information from base types as well when interfaces extend other interfaces.

Reply:
The paka thing doesn't work though for XState beta. Wanted to update it here for consistency but I guess we can keep it.